test cases/common/149 special characters/meson.build

```diff
@@ -0,0 +1,13 @@
+project('ninja special characters' ,'c')
+
+python = import('python3').find_python()
+
+ret = run_command(python, ['check_quoting.py', 'newline=
+', 'dollar=$', 'colon=:', 'space= '])
```

Comment:
We should also check that multiple newlines and spaces are also preserved as-is.